docs/architecture/cloud-native/relational-vs-nosql-data.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Relational vs. NoSQL data
+description: Learn about relational and NoSQL data in cloud-native applications
+author: robvet
+ms.date: 12/30/2019
+---
+# Relational vs. NoSQL data
+
+[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+
+## Introduction
+
+Relational and NoSQL are two types of database systems commonly implemented in cloud-native apps. They're built differently, store data differently, and accessed differently. In this section, we'll look at both. Later in this module, we'll look at an emerging database technology called *NewSQL*.
```

Comment:
module? what is a module in this context?

Reply:
Fixed. Change module to chapter. Thx